requirements.txt

```diff
@@ -0,0 +1,7 @@
+openai==1.65.3
```

Comment:
there were previous attempts to add requirements.txt . It's not here on purpose. Please read my older older comments on that PRs